src/PreCharge/GFDB.cpp

```diff
@@ -0,0 +1,117 @@
+#include <PreCharge/GFDB.hpp>
+
+
+namespace IO = EVT::core::IO;
+namespace time = EVT::core::time;
+
+namespace GFDB {
+
+    GFDB::GFDB(IO::CAN& can) : can(can){};
+
+    IO::CAN::CANStatus GFDB::init() {
+        return can.connect();
+    }
+
+    IO::CAN::CANStatus GFDB::requestVnHighRes(int32_t* highRes) {
+        uint8_t rxBuffer[8] = {};
```

Comment:
```suggestion
        uint8_t rxBuffer[8];
```

Also, why is this array 8 uint8_ts long? Aren't you only using 4?